### PR TITLE
{bio}[foss/2019b] MedPy v0.4.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by orbsmiv**
Depends on ~~#9747~~
... which provides:
- ITK-5.0.1-foss-2019b-Python-3.7.4.eb
- SimpleITK-1.2.4-foss-2019b-Python-3.7.4.eb

(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    (name, version, {
```
Comment (boegel) [inline easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb]: @orbsmiv If there's only a single extension, you might as well use `PythonPackage` (and then not use `exts_list`)?

File: easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description = """MedPy is a library and script collection for medical image processing in Python, providing basic
+ functionalities for reading, writing and manipulating large images of arbitrary dimensionality. Its main
+ contributions are n-dimensional versions of popular image filters, a collection of image feature extractors, ready
+ to be used with scikit-learn, and an exhaustive n-dimensional graph-cut package."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ("SimpleITK", "1.2.4", versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb]: We usually use single quotes for string values that don't have spaces (see line above)

File: easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (orbsmiv) [reply easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb]: Fixed in 2de401f

**Comment by orbsmiv** [comment]
Test report by @orbsmiv
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f4452dee5f359539c3c8858b53533cfd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/45f68ee32842150f9b8a865baa983348 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/719f3c9ac2cddac334017a9a951dab48 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3203.victini.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/5a3fa04a649169a44cb6e65f5462cdc3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @orbsmiv!

**Pull request merged**